Week2/exercise/scripts.js

```diff
@@ -0,0 +1,120 @@
+"use strict";
+
+// Array exercises
+// Write a JavaScript function to get the first element of an array. Passing a parameter 'n' will return the first 'n' elements of the array.
+
+const anArray = [
+    "the_vegetarian",
+    "watership_down",
+    "adventures_sherlock_holmes",
+    "alchemist",
+    "frankenstein",
+    "city_of_thieves",
+    "gone_with_the_wind",
+    "to_kill_mocking_bird",
+    "war_and_peace",
+    "things_fall_apart"
+];
+
+function get1stElement(arr, val) {
```

Comment:
still needs a default parameter for when you pass no val, then it should return the first element